ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1335,7 +1335,7 @@ void GCS_MAVLINK_Copter::handleMessage(mavlink_message_t* msg)
         mavlink_msg_set_attitude_target_decode(msg, &packet);
 
         // exit if vehicle is not in Guided mode or Auto-Guided mode
-        if ((copter.control_mode != GUIDED) && (copter.control_mode != GUIDED_NOGPS) && !(copter.control_mode == AUTO && copter.flightmode_auto.mode() == Auto_NavGuided)) {
```

Comment:
How about a `flightmode->accepts_guided_set_angle()` call here?

Reply:
sure, that seems OK.  I think we can do that in a separate PR 'cuz this PR is really just a renaming exercise.